Paper.tex

```diff
@@ -1047,7 +1122,7 @@ \subsection{Execution Environment}\linkdest{exec_env}
 \begin{itemize}
```

Comment:
Note that the execution environment needs to contain the base fee, and it does by virtue of containing the block header I__H, so no update is needed to the list to include the base fee.